### ISSUE TITLE
Unsupported EventIO Types

### ISSUE THREAD

**Issue by maxnoe** [cta-observatory/pyeventio#179]
TL;DR:

- [x] 1206    Camera Layout (never used)
- [x] 1216    ATMPROF See #186 
- [x] 1207     Trigger Time (never used)
- [ ] 2017     Pixel Calibriation (might be written by `read_cta --calibrate`
- [x] 2024     Run Statistics (Not used, could be added but not really needed)
- [x]  2025     MC Run Statisitics (Not used, could be added but not really needed)
- [ ] 2029     Auxiliary Digital Traces (For debugging with `-DWITH_AUX_TRACES`)
- [ ] 2030     Auxiliary Analog Traces (For debugging with `-DWITH_AUX_TRACES`)
- [x] 2031     FSPhot (Not implemented)

I asked Konrad to point me to how we could go about implementing support for the types we don't currently support.

Find his answer below (translated from German by me).

## 1206: CORSIKA layout

> The 1206 block would in principal be written by `write_camera_layout`, but this method 
is called nowhere, not even in the private IACT Interface Version `iact_full5.c`
The definitions haven't changed sind the move from SSCS to CVS in 2001.
Even in the prähistoric HEGRA code base, there is no hint of `write_camera_layout` and `IO_TYPE_MC_LAYOUT`.

## 1207:CORSIKA trigger time

> There is not even a method to write this container. So, forget both. 

## 2017 

`  IO_TYPE_HESS_PIXELCALIB,  write_hess_pixcalib(),   (Calibrated pixel intensities)`

> Container 2017 is not written by `sim_telarray` itself, but by `read_cta` if the `--calibrate` or the `--only-calibrated` options are used and an outputfile is written.

## 2024 and 2025

> The 2024 and 2025 were originally planned but never really neede and in the end not written.
One could add `write_hess_run_stat` and `write_hess_mc_run_stat` before writing the histogram block. But the histogram block contains more information so these are obvously not needed so there are no plans to implement them.

## 2029 to 2030

> These containers were defined in August 2018.

### 2029 and 2030
> 2029 and 2030 are written if sim_telarray is compiled with `-DWITH_AUX_TRACES` for debugging or illustrating intermediate steps. This would also need the `AUX_TRACES` Parameter at runtime, which is undocumented and uses 8 Integer values either 0 or 1 for the different traces.

### 2031

> Not implemented and not used

This would let me conclude, that for the moment, we are feature complete :fireworks:

**Comment by MaxNoe** [comment]
There is a new type, 1216, See #186
